content/Arduino Cloud/Arduino Chrome App/Use-Arduino-with-Chromebook.md

```diff
@@ -65,21 +60,13 @@ Follow these steps:
 
 1. Connect a [compatible board](https://support.arduino.cc/hc/en-us/articles/360014779899#chromebook) to one of the USB ports on your Chromebook.
 
-1. In the board selector menu, choose **Select Other Board & Port**
-
-   ![The Boards Dropdown in the Cloud Editor.](img/web-editor-boards-dropdown.png)
```

Comment:
`img/web-editor-boards-dropdown.png` is now unused so the file should be removed from the repository.